index-scheduler/src/index_mapper.rs

```diff
@@ -171,6 +186,87 @@ impl IndexMapper {
         Ok(self.index_mapping.get(rtxn, name)?.is_some())
     }
 
+    /// Resizes the maximum size of the specified index to the double of its current maximum size.
+    ///
+    /// This operation involves closing the underlying environment and so can take a long time to complete.
```

Comment:
Can you add one of the main reasons why it can take time, which is: search requests are running and must stop before we can close it, please?